_posts/2023-10-17-rust-in-yocto.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Rust on Yocto: A Seamless Integration"
+description: "A look at the various ways to integrate Rust into Yocto"
+author: tsarlandie
+tags: [rust, yocto, linux, embedded, cross-compilation]
+---
+
+<!-- excerpt start -->
+
+At Memfault, our love affair with Rust began in late 2022. What drew us to Rust? Well, the typical allure of a modern programming language: an impressive type-system, memory safety without the constant jitters, efficient concurrency management, a thriving package ecosystem, and overwhelming support from our engineering team. To put it simply, we're smitten. Our journey with Rust has been nothing short of transformative, enabling rapid progress and leading us to conquer challenges we previously deemed ... non-trivial.
```

Comment:
This excerpt is on the long side. I would suggest limiting it to just this first paragraph.